src/main/java/uk/gov/justice/hmpps/prison/service/FinanceService.java

```diff
@@ -111,4 +128,17 @@ private void validateTransferToSavings(final String prisonId, final String offen
             throw new ValidationException(String.format("Not enough money in offender sub account balance - %s", balance.setScale(2, RoundingMode.HALF_UP)));
         }
     }
+
+    private static final Function<Pair<Account, BigDecimal>, Account> setDamageObligation = (pair) -> {
+        pair.getFirst().setDamageObligations(pair.getSecond());
+        return pair.getFirst();
+    };
+
+    private static final Function<List<OffenderDamageObligationModel>, BigDecimal> sumAmountToPay = (obligations) -> obligations
+        .stream()
+        .map(OffenderDamageObligationModel::getAmountToPay)
+        .reduce(BigDecimal.ZERO, BigDecimal::add);
+
+    private static final Function<OffenderDamageObligationService, Function<String, BigDecimal>> createSumDamageObligationToPay =
+        (service) -> (Function<String, BigDecimal>) offenderNo -> sumAmountToPay.apply(service.getDamageObligations(offenderNo,""));
```

Comment:
should this include all status types?

Reply:
That's a good question. Should we only be counting active damage obligations?

Reply:
done